src/Directory.Build.props

```diff
@@ -5,4 +5,8 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageIcon>logo.64x64.png</PackageIcon>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <IsNetCore3OnwardsTarget>$(TargetFramework.StartsWith('netcoreapp3.'))</IsNetCore3OnwardsTarget>
```

Comment:
Does it work? I think that properties from Directory.Build.props are imported before TargetFramework defined in csproj.

Reply:
I think that's what I was doing over in the server sln, but definitely worth checking. It seemed to pull in the right references in the Tests project I used it in.